checkov/main.py

```diff
@@ -129,27 +129,34 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
         print_checks(framework=config.framework)
         return
 
+    baseline = None
+    if config.baseline:
+        baseline = Baseline.from_json(config.baseline)
+
     external_checks_dir = get_external_checks_dir(config)
     url = None
-
+    created_baseline_path = None
```

Comment:
@tronxd Why is it needed here?

Reply:
To pass it to runner/registry, if it's None nothing happens